pyproject.toml

```diff
@@ -21,17 +21,15 @@ dependencies = [
     "eth-typing",
     "hypothesis",
     "pytest",
-#'eth-stdlib; python_version >= "3.10"',
+    "rich",
```

Comment:
should this be an extra?

Reply:
I think it's fine. In fact, we should consider using rich in more places (because it is such an awesome module for prettifying console outputs).

It doesn't really bloat up boa, and is only imported at the end of the pytest session.